features/networking/pod.feature

```diff
@@ -190,3 +190,82 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-19810
```

Comment:
> I doubt that as it's written will work with 4.0 since it's accessing the hosts...which in 4.0 the external IP is deprecated.

I agree, this might need some tweak to support 4.0. Will look into that

Reply:
Hi @pruan-rht 
What is the criteria that we should put the case in public or tierN?

Reply:
@bmeng all private tests should be under there according to the [README](https://github.com/openshift/verification-tests/blob/3506c0863277bc75445526ea92062e052c8cd84a/README.adoc#running-tests-from-outside-this-repository)

Reply:
Ok. Thanks.

I do not think this case should be private testing. Let's keep it here.